docs/software_setup/docker.md

```diff
@@ -3,11 +3,43 @@
 In addition to the pre-built SD card image, we also provide a docker container
 in case you want to install JetBot on an existing Jetson Nano SD card.
 
-> For this, we'll assume you've set up your Jetson Nano using the **online Getting Started guide**.
-> 
-> - [Getting Started With Jetson Nano Developer Kit](https://developer.nvidia.com/embedded/learn/get-started-jetson-nano-devkit)
+???+ info
```

Comment:
Fixed

Reply:
I still see this admonition.  Unresolving since it's unchanged.

Let's remove this admonition, it doesn't seem necessary since this page is referenced from the "DIY JetBot" section in getting started guide.